DeAceroDLabs/marketplace-frontend
# Fix/dmp 506 Aligned signup buttons

## What is done

[[DMP-506] Navigation arrows on categories](https://deacero.atlassian.net/browse/DMP-506)

## How is done

- Updated `MultipleForm` styles for next and back buttons.

## How to test

1. Run the project via `npm start`
2. Click on `registrarse`.
3. You should see the buttons aligned correctly.

![image](https://user-images.githubusercontent.com/81311646/182271474-ae4f6e26-fae3-4926-a098-723cd5a4c8a1.png)
